src/main/java/com/openlattice/shuttle/transformations/Transformation.java

```diff
@@ -58,23 +59,19 @@ public String getColumn() {
     }
 
     protected String getInputString( Object o, Optional<String> column ) {
-        final String input;
         if ( o == null ) {
             return null;
         }
         if ( !( column.isPresent() ) ) {
-            input = o.toString();
-        } else {
-            ObjectMapper m = new ObjectMapper();
-            Map<String, String> row = m.convertValue( o, Map.class );
-            String col = getColumn();
-            if ( !( row.containsKey( col ) ) ) {
-                throw new IllegalStateException( String.format( "The column %s is not found.", column ) );
-            }
-            input = row.get( col );
+            return o.toString();
         }
-
-        return input;
+        ObjectMapper m = ObjectMappers.getJsonMapper();
+        Map<String, String> row = m.convertValue( o, Map.class );
+        String col = getColumn();
+        if ( !( row.containsKey( col ) ) ) {
```

Comment:
0% important but in the spirit of cleanup, can you change this to `if ( !row.containsKey( col ) ) {` ? I feel like the extra set of parenthesis makes this a bit harder to read (same for like 65 / other places this pattern is present)

Reply:
Haha, gladly!